api/openapi/v1/core-metadata.yaml

```diff
@@ -423,16 +386,13 @@ paths:
         200:
           description: JSON payload representing the device
         400:
-          description: for incorrect or unparsable requests
+          description: For incorrect or unparsable requests
         404:
-          description: device was not found
+          description: Device was not found
   /v1/device/id/{id}:
     delete:
-      description: Remove the Device designated by database generated id. This does
-        not remove associated objects (addressable, service, profile, etc.). Returns
-        Internal Service Error (HTTP 500) for unknown or unanticipated issues. Returns
-        NotFoundException (HTTP 404) if the device cannot be found by the identifier
-        provided.
+      description: Remove the device designated by database-generated ID. This does
```

Comment:
Another hyphen, please ctrl F the file and find and remove all instances of "database-generated"